src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -44,12 +43,14 @@ internal sealed partial class HttpConnection : IDisposable
         private static readonly byte[] s_spaceHttp11NewlineAsciiBytes = Encoding.ASCII.GetBytes(" HTTP/1.1\r\n");
         private static readonly byte[] s_hostKeyAndSeparator = Encoding.ASCII.GetBytes(HttpKnownHeaderNames.Host + ": ");
         private static readonly byte[] s_httpSchemeAndDelimiter = Encoding.ASCII.GetBytes(Uri.UriSchemeHttp + Uri.SchemeDelimiter);
+        private static readonly string s_cancellationMessage = new OperationCanceledException().Message;
```

Comment:
Should this be in the resource?

I'm not sure what will get returned here by default.  Probably a comment is warranted.

Reply:
I can copy the string to a resource.  Figured this was a straightforward way to just reuse the original message, but not a problem.

Reply:
I don't care either way, just wasn't sure what was going on here.  A comment would be fine.